deployment-template/ecs/aws-otel-ec2-sidecar-deployment-cfn.yaml

```diff
@@ -135,11 +127,31 @@ Resources:
           EntryPoint:
             - "/bin/sh"
             - "-c"
-            - "while true; do echo 'statsdTestMetric:1|c' | socat -v -t 0 - UDP:127.0.0.1:8125; sleep 1; done"
-      Memory: '2048'
-      Cpu: '1024'
+            - "while true; do echo 'statsdTestMetric:1|c' | socat -v -t 0 - UDP:aws-collector:8125; sleep 1; done"
+          Links:
+            - aws-collector
+        - Name: traffic-generator-xray
```

Comment:
I don't think adding this traffic generator is a good idea in it's current form. This will continue to infinitely generate metrics. The cost that is incurred by these metrics is non zero and may catch an end user off guard. I think we should either prevent this from running infinitely or exclude it all together while providing instruction on how to query the sample app.

Reply:
`traffic-generator-xray` is used to generate some traces from the `aws-xray-data-emitter`. Similar setup is used for metrics generation in the existing setup of `aoc-statsd-emitter`.  I am unsure of the proposal here: should I provide a timed exit from the while loop for the trace generation only or both trace and metric generation?